tutor/src/screens/gradebook/aggregate-result-cell.js

```diff
@@ -1,16 +1,31 @@
 import { React, PropTypes } from 'vendor';
+import { observer } from 'mobx-react';
+import { sumBy, filter } from 'lodash';
 import { getCell } from './styles';
+import S from '../../helpers/string';
+
 
 const Cell = getCell('0 10px');
 
+const getPoints = (tasks) => {
+  const aggregatePoints = sumBy(tasks, (t) => t.correct_exercise_count);
+  return S.numberWithOneDecimalPlace(aggregatePoints/tasks.length);
+};
+
+const getPercentage = (data) => {
+  return data ? `${S.asPercent(data.average_score)}%` : '100';
+};
```

Comment:
how about adding these to the task model?  that way the'd be avail to other code that might need them